tests/runtime/main.rs

```diff
@@ -97,13 +130,15 @@ fn tests(name: &str, dir_name: &str) -> Result<Vec<PathBuf>> {
     let mut c = Vec::new();
     let mut java = Vec::new();
     let mut go = Vec::new();
+    let mut c_sharp = Vec::new();
     for file in dir.read_dir()? {
         let path = file?.path();
         match path.extension().and_then(|s| s.to_str()) {
             Some("c") => c.push(path),
             Some("java") => java.push(path),
             Some("rs") => rust.push(path),
             Some("go") => go.push(path),
+            Some("cs") => c_sharp.push(path),
```

Comment:
how should we distingiush between cs llvm vs mono should that be here or a feature flag?

Reply:
I think just one `cs` but 2 features.